### ISSUE TITLE
callr name conflict

### ISSUE THREAD

**Issue by dfalster** [traitecoevo/callr#2]
Another package has recently appeared on cran with the name [callr](https://cran.r-project.org/web/packages/callr/index.html), so we'll need to consider renaming.

**Comment by richfitz** [comment]
I'm not 100% sure that callr will be the right thing to use going foward.  Gabor's processx will likely do the R->system link more nicely (fingers crossed) and I can try and port anything missing over there.

Not sure if anyone is using the reverse any more (I wrote it for Simon before the REST API was implemented) but if that's still useful we can rename and continue.

**Comment by dfalster** [comment]
Ok, thanks for the update. 

We use currently use callr in nearly all of our papers to make system calls, especially for compiling latex docs [e.g.](https://github.com/traitecoevo/plant_paper/blob/77420fba21ff91188e38b3efb31756d1c7a64c10/R/ms.R#L86)

While `processx` gets sorted, should I remove the callr dependency and instead add the call_system code to those projects?

**Comment by richfitz** [comment]
Yeah, that seems like a good fix. Or we can rename the package to avoid the clash and update the projects?
